tests/testthat/testCombine.R

```diff
@@ -0,0 +1,34 @@
+# testCombine.R
```

Comment:
- use the setup/teardown headers from ```tests/testthat/testDummy.R```
- always turn logging off, except for one block in which you explicitly test logging
- use ```tempdir()``` or ```tempfile()``` for intermediate files to be created, and delete them after testing.